Changelog.md

```diff
@@ -9,6 +9,7 @@ Bugfixes:
  * Standard JSON: Populate the ``sourceLocation`` field in the error list.
  * Type Checker: Suggest the experimental ABI encoder if using ``struct``s as function parameters
    (instead of an internal compiler error).
+ * Type Checker: Improve error message for wrong struct initialization.
```

Comment:
Nitpicking: this would be a feature? :)

Reply:
It’s under the Bugfixes title.

Reply:
I meant its more like a feature, but it doesn't really makes much of a difference.

Reply:
Yeah, not worth it.